pages/home/index.tsx

```diff
@@ -1,29 +1,40 @@
 import fs from 'fs'
 import path from 'path'
+import highlight from '@mapbox/rehype-prism'
+import rehypeSurfaceCodeNewlines from '@hashicorp/platform-code-highlighting/rehype-surface-code-newlines'
 import { serialize } from 'next-mdx-remote/serialize'
 import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote'
 import { GetStaticPropsResult } from 'next'
-import markdownDefaults from '@hashicorp/platform-markdown-utils'
```

Comment:
Our markdown defaults aren't fully compatible with mdx V2. So I imported the plugins we need directly.